src/NossoCalendario.Domain/Entities/Agenda.cs

```diff
@@ -0,0 +1,20 @@
+﻿using NossoCalendario.Domain.Base;
+using System;
+
+namespace NossoCalendario.Domain.Entities
+{
+    public class Agenda : Entity
+    {
+        public Agenda(string nome, string descricao, Guid usuarioId)
```

Comment:
Acho legal você receber o id do usuário como argumento. O que você precisa ficar preparado é que talvez o mercado fale que sua Agenda está relacionada como um usuário e não com um id de usuário :).

Reply:
Para receber diretamente o Usuario será necessário adicionar um construtor sem parâmetros. No mais não aferia o que já está implementado.

Reply:
Um construtor sem argumetos onde? No usuário? Se for isso, tudo bem. É uma restrição imposta pelo framework... Basta deixar uma dica deprecando aquele uso.

Reply:
Na classe Agenda. Já adicionei o atributo para não permitir a utilização do construtor em questão.